### DISCUSSION TITLE
is there a simple way to use http listener to act as a agent of an existing web server/service?

### DISCUSSION THREAD

**Discussion by linxd** [nextgenhealthcare/connect#5248] [Feature/Enhancement Ideas]
I want using mirth to act as a middler of a http service (just like api gateway ). how to do?

Thanks.

**Comment by jonbartels** [comment]
Just set up an HTTP listener and an HTTP sender. 

You'll want to pay close attention to headers. You will need to proxy the headers and the http method from the listener to the sender

**Reply by linxd** [reply]
Thanks 
  but  like a website.there are many redirect link , sublink will do much work.

**Reply by jonbartels** [reply]
Correct.

Mirth Connect *can* act as an HTTP proxy, but that does not mean it is the best tool for the job.
